### DISCUSSION TITLE
Introducing: Requests For Gitney!

### DISCUSSION THREAD

**Discussion by git900000** [nikoniko78/scripts-java-physion#35] [Scene Requests]
Hello there everyone! I am Gitney. I have created a repository; go to the Discussion tab! while you're there!
[Here!](https://github.com/git900000/Requests-For-Gitney)

Ask @nikoniko78 for more details about me, and/or what I do! ;)

**Comment by git900000** [comment]
And my pronouns are either they/them or he/him. Just so ya don't get mixed up.

**Reply by git900000** [reply]
(Who keeps upvoting everything I write!?)

**Reply by nikoniko78** [reply]
you ya goober its automatic

**Reply by git900000** [reply]
Oh... Huh!

**Comment by nikoniko78** [comment]
Please see the Scene Requests tab. You can post things like this there!

**Reply by nikoniko78** [reply]
p.s. The reason I am saying this is because I would like things related to this repository to stay localized. (please don't hate me)

**Reply by git900000** [reply]
I won't! Don't worry. Should I repost it in there?

**Comment by git900000** [comment]
Sure thing. And I have a question: Are you inviting others on Physion to this repository?
________________________________

**Reply by nikoniko78** [reply]
Yes, but not as advertising! I want everyone to stay connected and have easy access to scripts.

**Reply by git900000** [reply]
Ok!

**Comment by nikoniko78** [comment]
Hello Gitney! I see you have created a branch adding a script called Ragdoll creator. You can send a pull request to merge it to IndevScripts if you please (this is highly recommended)

**Comment by git900000** [comment]
I remember doing that though... 😕 I'll check again.
________________________________

**Reply by nikoniko78** [reply]
Your problem seems to be that you never actually added the script to your fork, so it will not do any change.

**Comment by git900000** [comment]
Alright.
________________________________
